class/features.php

```diff
@@ -0,0 +1,307 @@
+ * 
+ * @method validImage().
+ *   Checks wether given image is jpg/png and under 500kb.
+ * 
+ * @method splitMarks().
+ *   Splits marks and remove '|' and return array of marks.
+ * 
+ * @method validPhoneNo().
+ *   Checks if phone no is valid or not.
+ * 
+ * @method validMailId1().
+ *   Checks if Mail Id is valid or not using RegEx.
+ * 
+ * @method validMailBox().
+ *   Checks if Mail Id is valid or not using MailBoxLayer and Guzzle.
+ * 
+ * @method sendMail().
+ *   sends mail to given email Id.
+ * 
+ * @method getURL().
+ *   Get response body of given URL using GuzzleHTTP client request.
+ * 
+ * @property string $name
+ *  Store name off the user.
+ * 
+ * @property string $mailId
+ *  Store mailId of the user.
+ * 
+ * @property string $marks
+ *  Store marks of the user.
+ * 
+ * @property string $phoneNo
+ *  Store phoneNo of the user.
+ * 
+ * @property string $imagePath
+ *  Store path of profile-pic of the user.
+ * 
+ **/
+class features
+{
+    public $name;
+    public $mailId;
+    public $marks;
+    public $phoneNo;
+    public $imagePath;
+
+    // String methods here 
+
+    /** 
+     * Checks if a string only contains alphabets and whitespaces
+     * 
+     * @param  $string
+     *   stores the string to varify. 
+     **/
+    function onlyAlpha($string)
+    {
+        if (preg_match("/^[a-zA-Z-' ]*$/", $string)) {
+            return TRUE;
+        } 
+        else {
+            return FALSE;
+        }
+    }
+
+    /** 
+     * Fucntion to check the string only has digits
+     * 
+     * @param  $string
+     *   stores the string to varify. 
+     **/
+    function onlyDigit($string)
+    {
+        if (preg_match("/^[1-9][0-9]{0,15}$/", $string))
+            return TRUE;
+        else
+            return FALSE;
+    }
+
+    // Image methods here
+    /** 
+     *   Checks wether given image is jpg/png and under 500kb.
+     * 
+     * @param  $imageSize
+     *   Stores the size of the image. 
+     * 
+     * @param  $imageType
+     *   Stores the datatype of the image. 
+     **/
+    function validImage($imageSize, $imageType)
+    {
+        if (($imageSize / 1000) <= 500 && ($imageType == 'image/jpg' || $imageType == 'image/png' || $imageType == 'image/jpeg')) {
+            return TRUE;
+        } 
+        else {
+            if (($imageSize / 1000) > 500) {
+                echo "<br>Image size should be less than 500KB (" . ($imageSize / 1000) . "KB given)";
+            }
+            if ($imageType != 'image/jpg' || $imageType != 'image/png' || $imageType != 'image/jpeg') {
+                echo "<br>Only Jpeg, Jpg & Png are allowed (" . $imageType . " given)";
+            }
+            return FALSE;
+        }
+
+    }
+
+    /** 
+     * Splits the $marks string and return array of different strings
+     * 
+     * @param  $marks
+     *   Stores the marks field data of the user. 
+     * 
+     **/
+    function splitMarks($marks)
+    {
+        $lines = array();
```

Comment:
initialise the array like this:
```suggestion
        $lines = [];
```